packages/get-display-media/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@perry/get-media-display",
```

Comment:
let's try to keep the names consistent with the folders too, maybe we should rename it to `@perry/get-display-media`